SocietalConstructionTool/Runtime/QueryPredicate.cs

```diff
@@ -1,18 +1,11 @@
 namespace Sct.Runtime
 {
-    public class QueryPredicate : IQueryPredicate
+    public class QueryPredicate(string className, string? state, IDictionary<string, dynamic> fields) : IQueryPredicate
     {
-        public string ClassName { get; }
-        public string? State { get; }
+        public string ClassName { get; } = className;
+        public string? State { get; } = state;
 
-        public IDictionary<string, dynamic> Fields { get; }
-
-        public QueryPredicate(string className, string? state, IDictionary<string, dynamic> fields)
-        {
-            ClassName = className;
-            State = state;
-            Fields = fields;
-        }
+        public IDictionary<string, dynamic> Fields { get; } = fields;
```

Comment:
Er der nogen grund til at disse bør være public udover at `IQueryPredicate` siger det?
Man kan sagtens slette dem helt og så blot køre `Test()` på det man får i constructoren jo.

Har ikke fjernet det, da jeg tænkte at det måske kunne have noget at sige hvis vi kan holde referencer til et predicate senere.

Reply:
Jeg tror godt at de bare kan blive private felter for nu, da vi ikke kan tilgå felter på objekter fra sproget